func/uk/gov/hmrc/vatapi/resources/ObligationsResourceSpec.scala

```diff
@@ -96,7 +98,51 @@ class ObligationsResourceSpec extends BaseFunctionalSpec {
         .when()
         .get(s"/$vrn/obligations?from=2017-01-01&to=2017-08-31&status=A")
         .thenAssertThat()
-        .statusIs(404)
+        .statusIs(NOT_FOUND)
+    }
+
+    "return code 400 when idNumber parameter is invalid" in {
+      given()
+        .stubAudit
+        .userIsFullyAuthorisedForTheResource
+        .des().obligations.obligationParamsFor(vrn, Errors.invalidIdNumber)
+        .when()
+        .get(s"/$vrn/obligations?from=2017-01-01&to=2017-08-31&status=A")
+        .thenAssertThat()
+        .statusIs(BAD_REQUEST)
+    }
+
+    "return code 400 when regime type parameter is invalid" in {
```

Comment:
Minor comment, test description states we return a 400 but we assert on INTERNAL_SERVER_ERROR